### PR TITLE
Add updated catalog of all available CRDs

### PR DESCRIPTION
**Pull request by sgayangi**

### CODE REVIEW

**Review by Krishanx92 — APPROVED** [review]

File: en/docs/catalogs/crd.md [hunk 1]
```diff
@@ -0,0 +1,4359 @@
...
+        <tr>
+            <td>
+                <code>in</code></br>
+                <em>
+                    string
+                </em>
+            </td>
+            <td>
+                <pre><code>In is to specify how the APIKey is passed to the request
+</code></pre>
```
Comment (ashera96) [inline en/docs/catalogs/crd.md:1390]: Shouldn't the in field be just a `<p>` tag like the name field below that?

image

File: en/docs/catalogs/crd.md [hunk 2]
```diff
@@ -0,0 +1,4359 @@
...
+    <tbody>
+        <tr>
+            <td>
+                <code>apiVersion</code></br>
+                string
+            </td>
+            <td>
+                <code>
+dp.wso2.com/v1alpha1
+</code>
```
Comment (ashera96) [inline en/docs/catalogs/crd.md:1152]: Shall we fix the indentation issue here under the description of apiVersion field? Let's check other places similar to this as well

image

File: en/docs/catalogs/crd.md (hunk 1 above)
Comment (sgayangi) [reply en/docs/catalogs/crd.md:1390]: Fixed in PR https://github.com/wso2/docs-apk/pull/584

File: en/docs/catalogs/crd.md (hunk 2 above)
Comment (sgayangi) [reply en/docs/catalogs/crd.md:1152]: Fixed in PRs https://github.com/wso2/docs-apk/pull/583 and https://github.com/wso2/docs-apk/pull/584

**Pull request merged**